development/image-builder/README.md

```diff
@@ -130,5 +130,6 @@ Usage of image-builder:
         Additional tag that the image will be tagged
   -variant string
         If variants.yaml file is present, define which variant should be built. If variants.yaml is not present, this flag will be ignored
-
+  -export-tags
```

Comment:
I cannot find a test-case for this flag. Can you include it in the tests?

Reply:
Sure, I will do that shortly